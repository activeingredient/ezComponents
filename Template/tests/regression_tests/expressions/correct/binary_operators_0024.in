{var $foo = 42, $bar = 101, $bar2 = 201, $arr = array( 'foo' => 401 )}
0737:[{debug_dump( !( $foo / 5 ) + 5 )}]
0738:[{debug_dump( !( $foo / 5 ) + -$foo )}]
0739:[{debug_dump( !( $foo / 5 ) - 5 )}]
0740:[{debug_dump( !( $foo / 5 ) - -$foo )}]
0741:[{debug_dump( !( $foo / 5 ) * 5 )}]
0742:[{debug_dump( !( $foo / 5 ) * -$foo )}]
0743:[{debug_dump( !( $foo / 5 ) / 5 )}]
0744:[{debug_dump( !( $foo / 5 ) / -$foo )}]
0745:[{debug_dump( !( $foo / 5 ) % 5 )}]
0746:[{debug_dump( !( $foo / 5 ) % -$foo )}]
0747:[{debug_dump( !( $foo / 5 ) == 5 )}]
0748:[{debug_dump( !( $foo / 5 ) == -$foo )}]
0749:[{debug_dump( !( $foo / 5 ) === 5 )}]
0750:[{debug_dump( !( $foo / 5 ) === -$foo )}]
0751:[{debug_dump( !( $foo / 5 ) != 5 )}]
0752:[{debug_dump( !( $foo / 5 ) != -$foo )}]
0753:[{debug_dump( !( $foo / 5 ) !== 5 )}]
0754:[{debug_dump( !( $foo / 5 ) !== -$foo )}]
0755:[{debug_dump( !( $foo / 5 ) < 5 )}]
0756:[{debug_dump( !( $foo / 5 ) < -$foo )}]
0757:[{debug_dump( !( $foo / 5 ) <= 5 )}]
0758:[{debug_dump( !( $foo / 5 ) <= -$foo )}]
0759:[{debug_dump( !( $foo / 5 ) > 5 )}]
0760:[{debug_dump( !( $foo / 5 ) > -$foo )}]
0761:[{debug_dump( !( $foo / 5 ) >= 5 )}]
0762:[{debug_dump( !( $foo / 5 ) >= -$foo )}]
0763:[{debug_dump( !( $foo / 5 ) && 5 )}]
0764:[{debug_dump( !( $foo / 5 ) && -$foo )}]
0765:[{debug_dump( !( $foo / 5 ) || 5 )}]
0766:[{debug_dump( !( $foo / 5 ) || -$foo )}]
0767:[{debug_dump( !( $foo / 5 ) . 5 )}]
0768:[{debug_dump( !( $foo / 5 ) . -$foo )}]
