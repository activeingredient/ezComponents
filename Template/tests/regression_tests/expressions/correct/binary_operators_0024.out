0737:[5]
0738:[-42]
0739:[-5]
0740:[42]
0741:[0]
0742:[0]
0743:[0]
0744:[0]
0745:[0]
0746:[0]
0747:[false]
0748:[false]
0749:[false]
0750:[false]
0751:[true]
0752:[true]
0753:[true]
0754:[true]
0755:[true]
0756:[true]
0757:[true]
0758:[true]
0759:[false]
0760:[false]
0761:[false]
0762:[false]
0763:[false]
0764:[false]
0765:[true]
0766:[true]
0767:['5']
0768:['-42']
